src/scripts/main.js

```diff
@@ -358,3 +358,24 @@ const people = [
 console.log(people); // you can remove it
 
 // write your code here
+// step 1 . 1
+for (const person of people) {
+  // find table.tbody
+  const table = document.querySelector('.dashboard').firstElementChild;
+  // create tr with 6 cells and inner text for them
+  const row = document.createElement('tr');
+  const personsName = (document.textContent = person.name);
```

Comment:
The assignment `document.textContent = person.name` is incorrect. You should create a new cell element and set its text content to `person.name` instead.